vm/chef/cookbooks/elasticsearch/files/elasticsearch

```diff
@@ -117,27 +168,31 @@ if [[ "${es_current_node}" == "${es_first_hostname}" ]]; then
   wait_for_port "${es_current_node}" 9200
 
   # Auto generate passwords for built-in users using CLI
-  autogenerate_passwords
+  autogenerate_passwords "${es_protocol}"
   es_default_password="$(get_autogenerated_password elastic)"
 
-  # Update passwords according deployment.
+  # Update passwords according deployment manager variables.
   # Change elastic user password.
   change_user_password "${es_current_node}" \
     elastic "${es_default_password}" \
-    elastic "${es_password}"
+    elastic "${es_password}" \
```

Comment:
imho better to define all those users (elastic, kibana_system, logstash_system) at the beginning of the script than have them hardcoded everywhere